src/tokens/TokenBalances.tsx

```diff
@@ -47,28 +47,29 @@ import fontStyles from 'src/styles/fonts'
 import { getShadowStyle, Shadow, Spacing } from 'src/styles/styles'
```

Comment:
probably not necessary to change now, but I guess we could just do this change in the new Assets screen, since this is anyway going to be deprecated as part of the assets redesign work. I can copy these changes to #4226